tvm/src/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -395,8 +333,14 @@ void Schedule::stream_to(const Tensor& target, Stage dest, Stage source,
       dest_status = -1;
     }
```

Comment:
What are these status for? What are the possible values? If there are only two values, we can use a bool instead.

Reply:
This variable was created for some other purposes. I forgot to rename it. Will fix it